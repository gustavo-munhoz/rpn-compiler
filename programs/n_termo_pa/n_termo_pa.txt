(10 0 +)
(5 0 +)
(3 0 +)
( (2 RES) ( ((3 RES) 1 -) (1 RES) * ) + )
( ( (4 RES) ( (3 RES) (1 RES) + ) * ) 2 | )