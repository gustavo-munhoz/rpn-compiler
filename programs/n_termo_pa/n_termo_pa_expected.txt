# Cálculo de n-ésimo termo da PA
## N = 10
## a0 = 5
## r = 3
=> Valor esperado: 185 (0x59C8)